CONTRIBUTING.md

```diff
@@ -8,24 +8,40 @@
   - remove the wrapping conditions for `installCRDs` from all CRDs yaml files (will remove it from `values.yaml` too)
     - IMPORTANT NOTE: keep it for the Cluster* CRDs, just align the conditions to only check `crds.createClusterExternalSecret`
       and `crds.createClusterSecretStore` respectively
-- helper templates
-  - copy over upstream `_helpers.tpl` as `_eso_helpers.tpl`
-- generic templates
-  - copy over templates
-- values.yaml
-  - remove `installCRDs: true`, it will be controlled by `giantswarm.crds.install` instead
-  - remove `crds` section, we install all resources with a few exceptions
-    - IMPORTANT NOTE: keep `crds.createClusterExternalSecret`
-    - IMPORTANT NOTE: keep `crds.createClusterSecretStore`
-  - moved `.image` under `.giantswarm.images` + remove `.image`
-  - moved `.certController.image` under `.giantswarm.images` + remove `.certController.image`
-  - moved `.webhook.image` under `.giantswarm.images` + remove `.webhook.image`
-  - same for all 3 `securityContext` and `resources`
-  - check / reset all the above fields to the Giant Swarm image, security context, resources, etc. in the templates
-    including, but not limited to:
-    - `helm/external-secrets/templates/deployment.yaml`
-    - `helm/external-secrets/templates/cert-controller-deployment.yaml`
-    - `helm/external-secrets/templates/webhook-deployment.yaml`
+- sync subtree with upstream (based on: https://handbook.giantswarm.io/product/managed-apps/dev-experience/git-subtree/)
+  - pull changes to https://github.com/giantswarm/external-secrets-copy
+    - create and push a branch from the target tag you want to upgrade to e.g. from `v0.7.0` as `tag-v0-7-0`
+  - ```bash
+    # If you don't already have it:
+    git remote add -f --no-tags upstream-copy git@github.com:giantswarm/external-secrets-copy.git
+    
+    # Split upstream subdir
+    git checkout -b tmp
+    git subtree add --prefix upstream-tmp upstream-copy tag-v0-7-0 --squash
```

Comment:
I don't get this part... why `subtree add` here? It's not what you really want to add to the repo, right? If that's true, it's enough to:
- git checkout <tag-you-want-to-take-from>
- git subtree split

That way you don't need that `tmp` branch at all, you take directly from the tag and there's no extra `subtree add` that is deleted later anyway. Does it make sense?

Reply:
Did it this way cos not pulling the tags. But if we do, then we don't the whole create a branch from the tag part above, right?

Reply:
Updated 👍🏻